presto-mysql/src/test/java/io/prestosql/plugin/mysql/DockerizedMySqlServer.java

```diff
@@ -48,6 +48,7 @@ public DockerizedMySqlServer()
                         "MYSQL_USER", MYSQL_USER,
                         "MYSQL_PASSWORD", MYSQL_PASSWORD,
                         "MYSQL_DATABASE", "tpch"),
+                ImmutableList.of(),
```

Comment:
nit:
change cmt msg `Allow to set capabilities for DockerContainer` to `Allow setting capabilities in DockerContainer`